### ISSUE TITLE
Body - Back: 2 pieces?

### ISSUE THREAD

**Issue by epersonae** [brennx0r/feelsbot-knit-project#4]
"We'll need two finished pieces of the following:"

Just one, right?

**Comment by brennx0r** [comment]
Yup - just need one back piece. I will correct this issue once the current pull request is merged.
